packages/tests-postman/index.js

```diff
@@ -128,7 +134,7 @@ function getOAuth2Config(name) {
  */
 async function getBearerToken(name) {
   const params = getOAuth2Config(name);
-  const res = await ky
+  return ky
```

Comment:
my await/async comment was more targetted at a line like this one, where it seems the change was made without any loops. @brownoxford